tasks/eth/base-003-holocene-fp-upgrade/input.json

```diff
@@ -0,0 +1,67 @@
+{
+  "chainId": 1,
+  "metadata": {
+    "name": "Base Mainnet Holocene - Proof Contract Upgrades",
+    "description": "Upgrades the `MIPS.sol`, `FaultDisputeGame.sol`, and `PermissionedDisputeGame.sol` contracts for Holocene."
+  },
+  "transactions": [
+    {
+      "metadata": {
+        "name": "Upgrade `PERMISSIONED_CANNON` game type in `DisputeGameFactory`",
+        "description": "Upgrades the `PERMISSIONED_CANNON` game type to the new Holocene deployment, with an updated version of `op-program` as the absolute prestate hash."
```

Comment:
```suggestion
        "description": "Upgrade the `PERMISSIONED_CANNON` game type to the new Holocene deployment, with an updated version of `op-program` as the absolute prestate hash."
```

Reply:
This just omits the `It` which is common in short descriptive sentences. Also, the `"description"` fields has no impact at all, so I'll just leave it as is, also because it's auto generated and I don't want to change the auto-generation script.